staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -226,10 +226,19 @@ type GroupResources struct {
 	// The empty string represents the core API group.
 	// +optional
 	Group string
-	// Resources is a list of resources within the API group. Subresources are
-	// matched using a "/" to indicate the subresource. For example, "pods/log"
-	// would match request to the log subresource of pods. The top level resource
-	// does not match subresources, "pods" doesn't match "pods/log".
+	// Resources is a list of resources this rule applies to.
+	//
+	// For example:
+	// 'pods' means pods.
+	// 'pods/log' means the log subresource of pods.
+	// '*' means all resources and their subresources.
+	// 'pods/*' means all subresources of pods.
+	// '*/scale' means all scale subresources.
+	//
+	// If wildcard is present, the validation rule will ensure resources do not
```

Comment:
> If wildcard is present, the validation rule will ensure resources do not overlap with each other.

A little confused by this statement. What's it trying to indicate?

Reply:
It means `Resources []string` elements should not indicate same resource.

Reply:
What's wrong with when they overlap?

Reply:
I don't see any validation you're talking about.

Reply:
Yeah I don't see this either. Can you drop this sentence?